src/gui/macutilities.mm

```diff
@@ -106,7 +106,14 @@ void openFiles(const PathList &pathList)
             for (const auto &path : pathList)
                 [pathURLs addObject:[NSURL fileURLWithPath:path.toString().toNSString()]];
 
-            [[NSWorkspace sharedWorkspace] activateFileViewerSelectingURLs:pathURLs];
+            // in some unknown way, the next line affects Qt's main loop causing the crash
+            // in QApplication::exec() on processing next event after this call.
+            // even crash doesn't happen exactly after this call, it will happen on
+            // application exit. call stack and disassembly are the same in all cases.
+            // but running it in another thread (aka in background) solves the issue.
+            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
What does this expression mean in Objective-C? I'm assuming it's lambda, right? Can it be reformatted to follow our coding style more? I mean move the opening brace to a separate line.

Reply:
yes, this is lambda.